docs/docsite/rst/contributor/index.rst

```diff
@@ -10,13 +10,14 @@ There are so many ways you can contribute to AWX.
 
 **Join us online**
 
-You can chat with us and ask questions on Matrix at `#awx:ansible.com <https://matrix.to/#/#awx:ansible.com>`_ or visit the `Ansible Community Forum <https://forum.ansible.com/c/project/7/>`_ to find contributor resources.
+Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide<communication>` to learn how to join the conversation!
```

Comment:
```suggestion
Need help or want to discuss AWX including the documentation? See the :ref:`Communication guide <communication>` to learn how to join the conversation!
```

Reply:
verified on the published docs that this seems to work with or without the spaces.